docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
 
 You either have:
 
-- tiered licensing, or
-- combined licensing
+- [combined licensing](#combined-licensing), or
+- [tiered licensing](#tiered-licensing-deprecated) (deprecated)
 
-To understand which license type you have, refer to [Determine the number of active users for each licensed role](#determine-the-number-of-active-users-for-each-licensed-role).
```

Comment:
Why is this line removed? This seems more relevant with the deprecation, so users can determine whether they need to update their licensing.

Reply:
This will no longer be available in 9.0. I guess we could replace this with "contact your Grafana account team"? cc @mjseaman

Reply:
Agree @AgnesToulet